hbase-client/src/main/java/org/apache/hadoop/hbase/client/Table.java

```diff
@@ -459,9 +459,10 @@ default CheckAndMutateResult checkAndMutate(CheckAndMutate checkAndMutate) throw
    * {@link Put} and {@link Delete} are supported.
    *
    * @param rm object that specifies the set of mutations to perform atomically
+   * @return results of Increment/Append operations
    * @throws IOException
```

Comment:
nits: could remove the empty throws so we could fix on checkstyle warning.

Reply:
Sure. I will do that.